### PR TITLE
Admin view

### PR DESCRIPTION
**Pull request by MoritzHahn1337**
-> Added Group and User Tabs in the Admin View
-> Added Possibilities to Change user admin & enabled settings
-> Added Possibilities to change groups
-> Added Possibilities to change user and group names

### CODE REVIEW

File: datameta/static/js/admin.js [hunk 1]
```diff
@@ -184,12 +185,382 @@ DataMeta.admin.reload = function() {
...
+        rowId: 'uuid',
+        order: [[1, "asc"]],
+        paging : true,
+        searching: false,
+        columns: [
+            { title: "User ID", data: "id"},
+            { title: "Name", data: "fullname", render:function(data) {
+                return '<button type="button" class="py-0 px-1 btn btn-sm enabled" onclick="changeUserName(event);" data="' + data + '">' + data + ' <i class="bi bi-pencil-square"></i></button>';
+            }},
+            { title: "eMail Adress", data: "email" },
```
Comment (lkuchenb) [inline datameta/static/js/admin.js:211]:
```suggestion
            { title: "Email Address", data: "email" },
```

**Review by lkuchenb — CHANGES_REQUESTED** [review]
Very nice, thanks! 👍 

One minor remark about datatables, in you don't have to destroy and rebuild the tables from scratch if you just want to update the data. You can define the datatable initially (w/o data) and upon refresh just do

```javascript
var t = $('#table_users').DataTable()
t.clear();
r.rows.add(newData);
r.draw();
```
That will be much more resource efficient for the browser. The reason why I'm not using this with the metadata tables is that in principle the metadata columns can change between updates (if in the meantime an admin changed the metadata definitions), thus I intentionally go for the full rebuild every time. This is not the case for the user / group etc.

You can update it if you like or keep it as it is, in anyway it may be good to know for future tables to come :)

**Comment by MoritzHahn1337** [comment]
Changed the table update

File: datameta/static/js/admin.js [hunk 2]
```diff
@@ -184,12 +185,398 @@ DataMeta.admin.reload = function() {
...
+        rowId: 'uuid',
+        order: [[1, "asc"]],
+        paging : true,
+        searching: false,
+        columns: [
+            { title: "User ID", data: "id"},
+            { title: "Name", data: "fullname", render:function(data) {
+                return '<button type="button" class="py-0 px-1 btn btn-sm enabled" onclick="changeUserName(event);" data="' + data + '">' + data + ' <i class="bi bi-pencil-square"></i></button>';
+            }},
+            { title: "Email Adress", data: "email" },
```
Comment (lkuchenb) [inline datameta/static/js/admin.js:218]:
```suggestion
            { title: "Email Address", data: "email" },
```
'address' is still misspelled 😉

**Review by lkuchenb — CHANGES_REQUESTED** [review]

File: datameta/static/js/admin.js (hunk 2 above)
Comment (lkuchenb) [reply datameta/static/js/admin.js:218]: Ah it's not the same location. Must have overlooked that one before.

**Review by lkuchenb — APPROVED** [review]

**Pull request merged**